source/recovery.rst

```diff
@@ -63,7 +63,22 @@ which is based on three PXC Pods.
 .. note:: The following commands are written for PXC 8.0. The same steps are
    also for PXC 5.7 unless specifically indicated otherwise.
 
-1. Change the normal PXC image inside the cluster object to the debug image:
+#. Check the current Update Strategy with the following command:
+
+   .. code-block:: bash
+
+      $ kubectl get pxc cluster1 -o jsonpath='{.spec.updateStrategy}'
+
+   If the returned value is different from ``onDelete`` (e.g., you are using
```

Comment:
I think RollingUpdate can also be ok, so I wouldn't say `is different from onDelete` but maybe just if it is SmartUpdate then change it to onDelete